dev/com.ibm.ws.beanvalidation.v20.cdi/src/com/ibm/ws/beanvalidation/v20/cdi/internal/ValidationReleasableFactoryImpl.java

```diff
@@ -39,13 +46,12 @@
 import com.ibm.ws.beanvalidation.service.ValidationReleasable;
 import com.ibm.ws.beanvalidation.service.ValidationReleasableFactory;
 import com.ibm.ws.cdi.CDIService;
-import com.ibm.ws.container.service.metadata.ComponentMetaDataListener;
-import com.ibm.ws.container.service.metadata.MetaDataEvent;
+import com.ibm.ws.cdi.internal.interfaces.CDIRuntime;
+import com.ibm.ws.kernel.service.util.PrivHelper;
```

Comment:
We're not supposed to use PrivHelper anymore, can you please remove the dependency and deprecate the PrivHelper class while you're at it?
@NottyCode has pointed out that although all liberty bundles run with the same permissions today, that may not hold true in the future.

Reply:
Done.